tests/link/pytorch/test_extra_ops.py

```diff
@@ -41,3 +41,50 @@ def test_pytorch_CumOp(axis, dtype):
         out = pt.cumprod(a, axis=axis)
         fgraph = FunctionGraph([a], [out])
         compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize("axis", [0, 1])
+def test_pytorch_Repeat(axis):
+    a = pt.matrix("a", dtype="float64")
+
+    test_value = np.arange(6, dtype="float64").reshape((3, 2))
+
+    out = pt.repeat(a, (1, 2, 3) if axis == 0 else (3, 3), axis=axis)
```

Comment:
Same here, do we allow `axis=None` like `numpy`? It may fail because we don't yet have Reshape implemented but we should test